### ISSUE TITLE
Update SSH-related info still mentioning `PLATFORM_RELATIONSHIPS`

### ISSUE THREAD

**Issue by AnouckColson** [platformsh/platformsh-docs#3837]
### Where on docs.platform.sh should be changed?

https://docs.upsun.com/development/ssh.html#connect-to-services
https://docs.upsun.com/get-started/stacks/symfony/local.html#2-create-the-tethered-connection

https://docs.upsun.com/development/local/tethered.html#create-the-tethered-connection
https://docs.upsun.com/languages/nodejs/debug.html#2-get-access

### What exactly should be updated?

Now that service environment variables have been documented via #3809 , we still haves places in the docs that mention the `PLATFORM_RELATIONSHIPS` variable in use cases involving the use of SSH.

The first 2 links are actually outputs from the CLI so no change might be needed => check with @lolautruche what, if anything, is planned on the CLI side to amend those outputs.

The other 2 links point to procedures that may need to be updated in order to feature service environment variables (instead of the `PLATFORM_RELATIONSHIP` variable), but some testing needs to be conducted. See with DevRel who wants to tackle what

**Comment by flovntp** [comment]
- [ ] do a pull request on CLI repo to add an example of how to display service environment variable
```
* Display relationships (use quotes for complex syntax):
```
upsun environment:ssh 'echo $PLATFORM_RELATIONSHIPS | base64 --decode'
```
```

**Comment by gilzow** [comment]
@AnouckColson  I dont think we can remove this completely as it is still _needed_ for tethered connections. 

If I'm attempting to create a tethered connection and have my local codebase connect to those connections, there is no way (currently) to _easily_ replicate those service environment variables locally. The easiest, most efficient method is to still use the $PLATFORM_RELATIONSHIPS | base64 --decode method in my .environment file, then after creating the tether, perform 
```
export PLATFORM_RELATIONSHIPS="$(upsun tunnel:info --encode)"
source .environment
```

Ideally the cli will be able to generate a file that creates the same service environment variables that can be sourced, but until then, those that use tethered connections *must* still use the above.
